docs/teams.md

```diff
@@ -0,0 +1,30 @@
+# Teams BETA
+
+A team in Quilt is a separate package namespace and access control specification. Being part of a team lets you install and push team-visible packages.
+
+The general notation for a team package is team:user/pkg.
+
+Most Quilt commands work just like normal whether you're part of a team or not. The rest of this document lists all the exceptions.
+
+## `quilt push`
+
+`quilt push --public` is disallowed for team packages. Use `quilt push --team` to push a package visible to everyone in your team.
+
+Example: `quilt push --team myteam:me/my_pkg`
+
+## `quilt access`
+
+Public visibility is disabled for team packages. To make a private team package visible to your team, run `quilt access add team:user/pkg team` where `team` is the name of your team. To make a private package visible to select other members of your team, run `quilt access add team:user/pkg other_user` for each user you want to add.
+
+Example: `quilt access add myteam:me/mypkg myteam` to make package team-visible.
+`quilt access add myteam:me/mypkg other_user_on_my_team` to share a private package.
+
+## `quilt login`
```

Comment:
Is there no command in the specs doc that's missing from here? Seems short. See table we worked on.